kong/plugins/oauth2/migrations/postgres.lua

```diff
@@ -164,5 +166,23 @@ return {
     down = [[
       ALTER TABLE oauth2_credentials ADD CONSTRAINT oauth2_credentials_client_secret_key UNIQUE(client_secret);
     ]],
+  },
+  {
+    name = "2017-10-11-oauth2_new_refresh_token_ttl_config_value",
+    up = function(_, _, dao)
+      for ok, config, update in plugin_config_iterator(dao, "oauth2") do
+        if not ok then
+          return config
+        end
+        if config.run_on_preflight == nil then
+          config.run_on_preflight = true
```

Comment:
too much copy-pasta here, `run_on_preflight` is not being added here but the config option `refresh_token_ttl`.

Reply:
🤕 oh my, Bob! Thanks